taxonomy-editor-frontend/src/App.js

```diff
@@ -0,0 +1,20 @@
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import Entry from "./pages/allentries";
+import EditEntry from "./pages/editentry";
+import Home from './pages/home';
+
+function App() {
+  return (
+    <Router>
+      <div className="App">
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/entry" element={<Entry />} />
+          <Route path="/entry/:id" element={<EditEntry />} />
+        </Routes>
+      </div>
+    </Router>
```

Comment:
I don't know if it can be moved easily later on (and in this case we can let it like that for now), but I think in the long run, we will have something like: "/:branch-name/:taxonomy/entry/:id"
- branch-name would be the name of a branch in git
- taxonomy would be taxonomy name (like categories)

Reply:
@alexgarel Yep, I think it would be possible. The "branch-name" property might be a little difficult to bring in right away, due to its complexity related to auth and error handling. The "taxonomy" property can be brought in, whenever the provision of seperate databases for each taxonomy is brought in Neo4J.